src/main/api/fileStore.ts

```diff
@@ -0,0 +1,79 @@
+import path from 'path'
+
+import { app, ipcRenderer, ipcMain } from 'electron'
+import * as fs from 'fs-extra'
+
+import { ApiFileStoreService, StoreFileData, StoreFileName } from '../../shared/api/types'
+import { getStoreFilesIPCMessages } from '../../shared/ipc/fileStore'
+
+const APP_NAME = app?.name ?? 'ASGARDEX'
+
+const APP_DATA_DIR = path.join(app?.getPath('appData') ?? './testdata', APP_NAME)
+export const STORAGE_DIR = path.join(APP_DATA_DIR, 'storage')
+
+// If `fullFilePathname' does not exist, `fs.remove` silently does nothing.
+// @see https://github.com/jprichardson/node-fs-extra/blob/master/docs/remove.md
+const removeFile = async (fullFilePathname: string) => fs.remove(fullFilePathname)
+
+const isFileExist = async (fullFilePathname: string) => fs.pathExists(fullFilePathname)
```

Comment:
^ Not the same, but very similar to [`removeKeystore` and `keystoreExist` ](https://github.com/thorchain/asgardex-electron/blob/e7beb5b78802c3b635c90ec8eb9eb1a76b3247e1/src/main/api/keystore.ts#L22-L24). Maybe we can bring it "together" by using `removeFile` / `isFileExists` in _keystore.ts_

Reply:
true. but i decided not to touch a code which is not supposed to be refactored at this issue for the sake of time and to keep PR smaller.  despite on the fact it's not a huge one it includes a lot of logical connections which should be kept in mind and changing previous implementation would increase that logic. lets try to generalize this things a little bit later

Reply:
Makes sense.